front/app/modules/commercial/insights/utils/getSelectedCategoryFilter.ts

```diff
@@ -0,0 +1,8 @@
+const getSelectedCategoryFilter = (categoryQuery?: string) =>
```

Comment:
Honest question, not really suggestion : Could and should this util get the query by itself and maybe take off some dependencies on the router for some components using it ?

Reply:
Interesting idea! We could turn it into a custom hook perhaps, wonder if this will really simplify things or not ... I've changed the method a bit in my other pr so I will think about it and maybe change it there. I hope that's ok with you :)